components/Sidebar.js

```diff
@@ -54,7 +30,6 @@ Sidebar.propTypes = {
 
 const StyledAside = styled.aside`
   flex: 1 0 342px /* large breakpoint / 3 */;
-  padding: 1em;
```

Comment:
This padding seems to be overridden when we wrap our content in a new `<div>`. It looks like removing it maintains the padding as it currently exists in production.